xml/System.Data/DataRowView.xml

```diff
@@ -706,6 +713,8 @@
   
```

Comment:
line 702

Change it to "The column index."

Reply:
@mairaw I suppose you mean the description for the "ndx" parameter? Line 702 does not match what you're asking to modify. I think the line numbers got messed up after the last batch of updates.